Source/Core/Common/SymbolDB.cpp

```diff
@@ -39,6 +41,32 @@ void SymbolDB::Index()
 	}
 }
 
+void SymbolDB::PopulateSymbolComments()
+{
+	std::string filename = File::GetUserPath(D_MAPS_IDX) + SConfig::GetInstance().m_strUniqueID + "_comments.txt";
+	if (!File::Exists(filename))
+		return;
+	std::fstream line_parse(filename);
+	std::string line;
+	while (std::getline(line_parse, line))
+	{
+		unsigned int address = 0;
+		if (line.empty())
+			continue;
+		AsciiToHex(line.substr(0,8),address);
```

Comment:
what about making 54: `if (line.empty())` to `if (line.length() <= 8)` to catch empty comments and lines without enough characters?